### ISSUE TITLE
Search results take you to wrong page

### ISSUE THREAD

**Issue by BarryWarrington** [NaturalHistoryMuseum/scratchpads2#6491]
The species pages on my website are displayed as shown below. The layout and appearance is spot on;

![2D1C6533-C7D6-4F45-B4FE-86620A55FC84](https://user-images.githubusercontent.com/75427252/145601834-7d36f4a0-c668-4b4d-afa0-c31152519ac5.png)

If you use the search ‘all’ box, the following page loads;

![AB6F64D7-0E39-4262-8775-6ABFC43B8927](https://user-images.githubusercontent.com/75427252/145602074-8928bb07-3356-447e-9524-fcef36988ada.png)

If you then click on the species (Aulagromyza heringii), it loads up a different format of the species page (compare to the first screenshot);

![B16DD6A4-684E-422D-86A6-F960217E35ED](https://user-images.githubusercontent.com/75427252/145602592-3199324c-6ba5-4aac-ba2e-dc214b851bd5.png)

This page format isn’t great. Is it possible so that the search results take you to the ‘proper’ species page and not this version?

**Comment by therobyouknow** [comment]
Good morning @BarryWarrington looking into this for you.

**Comment by therobyouknow** [comment]
From your comments on comparison, have I understood correctly that the left side bar is missing? The missing left side bar contains the hierarchical taxonomy - see my annotated screenshot below.  Is this correct in terms of what's missing?


![image](https://user-images.githubusercontent.com/1080646/145790756-8866989e-3a95-4753-9113-88906f024bf0.png)

**Comment by BarryWarrington** [comment]
If you look at my first screen shot, the headings (nomenclature, media, summary) are in green and all the relevant text is nicely spaced etc. On my third screen shot, the headings are missing and all the text is lumped together in boxes.

I think the search results take you to the third screen shot page format as in ‘structure’, then ‘taxonomy’, ‘GB&I’ species then ‘settings’, I don’t have the ‘descriptions’ tab ticked. This is by choice as I much prefer the ‘overview’ screen format (which is my first screenshot’).

**Comment by therobyouknow** [comment]
Thank you @BarryWarrington has this happened since the release? ( https://github.com/NaturalHistoryMuseum/scratchpads2/issues/6479 )

**Comment by BarryWarrington** [comment]
No, this happens since I first set the site up. If users use the search box, I would like it to take them to the first screen shot page, not the third (using the search function is the only way you can actually see the third screen shot page format).

**Comment by therobyouknow** [comment]
Thank you @BarryWarrington I will examine the data with a copy of the site in my development set up.
